feature/mypage/src/main/java/com/mument_android/mypage/MyPageViewModel.kt

```diff
@@ -34,6 +51,27 @@ class MyPageViewModel : ViewModel() {
             userImg = userImg.value ?: 0
         )
     }
+
+    //이유 선택 박스 눌렀을 때
+    fun clickReasonChooseBox() {
+        _isClickReasonChooseBox.value = _isClickReasonChooseBox.value != true
+    }
+
+    //이유 선택 하기
+    fun clickReasonChoose() {
+        _isClickReasonChoose.value = true
+    }
+
+    //탈퇴 동의
+    fun clickUnregisterAgree() {
+        _isUnregisterAgree.value = _isUnregisterAgree.value != true
```

Comment:
```suggestion
     _isUnregisterAgree.value = !(_isUnregisterAgree.value!!)
```
- 이렇게 하면 명시적으로 더 좋을듯!! 근데 왜 nullable한 데이터인지는 잘 모르겠넹

Reply:
_isClickReasonChooseBox도 동일합니다!!

Reply:
옹 일단 고쳐보겠습니당! 감사합니다.!